### ISSUE TITLE
Whether graph data augmentation is used？

### ISSUE THREAD

**Issue by Sun217** [yangzhao1230/GraphTextRetrieval#5]
Is the graph data augmentation mentioned in the methods section of the paper also used in the cross-modal retrieval task. I checked the files related to data loading and there doesn't seem to be any relevant code. It will be appreciated if you can answer my query.

**Comment by yangzhao1230** [comment]
We do not  use graph data augmentation in downstream tasks as it hurt the performance. You can find related code in out early commits like: https://github.com/ddz16/MoMu/commit/d70b938f5f8596e47d04697e2c5b90c3e3580468#diff-c1d7df23ff03a256d0d05c1d24f2c2bde9a000fca91831067377d30503092a90

**Comment by Sun217** [comment]
> We do not use graph data augmentation in downstream tasks as it hurt the performance. You can find related code in out early commits like: [ddz16/MoMu@d70b938#diff-c1d7df23ff03a256d0d05c1d24f2c2bde9a000fca91831067377d30503092a90](https://github.com/ddz16/MoMu/commit/d70b938f5f8596e47d04697e2c5b90c3e3580468#diff-c1d7df23ff03a256d0d05c1d24f2c2bde9a000fca91831067377d30503092a90)
Thank you very much for your answer, I have found the relevant code for graph data augmentation in the pre-training code. Thank you for telling me that using it in a downstream task would compromise accuracy.
